avm/src/vm/event.rs

```diff
@@ -8,21 +7,26 @@ use crate::vm::memory::HandlerMemory;
 use crate::vm::opcode::OpcodeFn;
 use crate::vm::program::Program;
 use crate::vm::run::EVENT_TX;
+use crate::vm::InstrType;
+use crate::vm::InvalidState;
+use crate::vm::VMError;
+use crate::vm::VMResult;
 
 pub const NOP_ID: i64 = i64::MIN;
 
 #[derive(PartialEq, Eq, Hash)]
+#[repr(i64)]
 /// Special events in alan found in standard library modules, @std.
 /// The IDs for built-in events are negative to avoid collision with positive, custom event IDs.
 /// The first hexadecimal byte of the ID in an integer is between 80 and FF
 /// The remaining 7 bytes can be used for ASCII-like values
 pub enum BuiltInEvents {
   /// Alan application start
   /// '"start"' in ASCII or 2273 7461 7274 22(80)
-  START,
+  START = -9213673853036498142,
```

Comment:
What's going on here?

Reply:
with this, we can use `BuiltInEvents::START as i64`